src/domain/runtime/event/mod.rs

```diff
@@ -1,27 +1,43 @@
 //![Runtime events](https://chromedevtools.github.io/devtools-protocol/tot/Runtime)
 
-use std::fmt;
-
+use crate::domain::runtime::r#type::RemoteObject;
```

Comment:
Huh, don't see raw idents used that often!

Reply:
I'm gonna be honest... that's totally me cargo culting the previous implementation. I assume that those are littered through this library just so that "type" could be used as a module. I'm not a fan of that structure, maybe we should refactor it if this crate needs more changes.

Reply:
Doesn't bother me, and certainly isn't related to this patch anyway.